pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -100,6 +100,12 @@ func fixDaemonTaskInstalled() error {
 }
 
 func removeDaemonTask() error {
+	// Return nil if the task is not even exist
```

Comment:
`does not exist`
When this code gets reworked, wrapping all the `powershell.Execute("Get-ScheduleTask....") in a small helper API would make this code easier to read, and remove the need for these comments (`TaskInstalled(taskName)`, `TaskRunning(taskName)`, ...)